runtime/interpreter/interpreter.go

```diff
@@ -2446,7 +2444,7 @@ func (interpreter *Interpreter) declareCompositeValue(
 
 			if declaration.CompositeKind == common.CompositeKindResource {
 				uuid := interpreter.uuidHandler()
-				fields[ResourceUUIDMemberName] = UInt64Value(uuid)
+				fields[sema.UUIDFieldName] = UInt64Value(uuid)
```

Comment:
Just curious about the UUID changes, looks like you are adding a predeclared `uuid` member also?

Reply:
Yes, I just saw that we were redeclaring a constant for the existing `uuid` field name: `ResourceUUIDMemberName`. It's better to have a single source of truth, which is `sema.UUIDFieldName`